### DISCUSSION TITLE
What is polyfill ؟

### DISCUSSION THREAD

**Discussion by MM25Zamanian** [AliMD/alwatr#68] [Q&A]
What is a polyfill and what does it do and what is its history?

> پلی فیل چیست و چه کاری انجام می دهد و تاریخچه آن چیست؟

**Comment by armanasadian** [comment]
> What is a polyfill and what does it do and what is its history?
> 
> > پلی فیل چیست و چه کاری انجام می دهد و تاریخچه آن چیست؟


please, answer to this question @AliMD?
